### DISCUSSION TITLE
my request to student benefits has been approved

### DISCUSSION THREAD

**Discussion by h4hany** [community/community#56303] [GitHub Education]
### Hi everyone,

my request to student benefits has been approved since 21/05/2023 and i got this message 
Congrats! Your request for [@h4hany](https://github.com/h4hany) was approved and your benefits will be available soon.
but till now i didn't get the benefits

**Comment by dongmt2** [comment]
I'm having the same problem. Did you get it.

**Reply by h4hany** [reply]
yes, it took about 3 days to 5 days

**Reply by arrehman3** [reply]
can you guide me on how to use the benefits or like what is the road map to start a course?

**Reply by Crysil25** [reply]
H4hany did you get your benefits?

**Comment by eslamalbaik** [comment]
i have same problem , i think waiting 24 hours - 3 days

**Reply by Johnny1919** [reply]
Mine almost 4 days but nothing happens

**Comment by khanabdul0** [comment]
> ### Hi everyone,
> my request to student benefits has been approved since 21/05/2023 and i got this message Congrats! Your request for [@h4hany](https://github.com/h4hany) was approved and your benefits will be available soon. but till now i didn't get the benefits
> 
> ### A suggestion...(optional)
> _No response_

same problem

**Comment by samus-aran** [comment]
Hey all 👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. **However, during busy periods like the start of a new semester, it may take a bit longer.**

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!
